crates/ursa-network/src/service.rs

```diff
@@ -675,16 +697,26 @@ where
                         )
                     }
                 }
-
-                println!("cosmos");
             }
             NetworkCommand::Put { cid, sender } => {
```

Comment:
is this NetworkCommand::Put supposed to be called by put_car api?
if not, where should it be called?